### PR TITLE
capitalise all usernames. Log and signout if error getting admin token

### PR DESCRIPTION
**Pull request by whitfield-mj**

### CODE REVIEW

File: server/authentication/authInit.js [hunk 1]
```diff
@@ -5,12 +5,13 @@ const logger = require('../../log');
 module.exports = function(signInService, userService, audit) {
 
     async function localInit(username, password, done) {
+        const capUsername = username.toUpperCase();
```
Comment (petergphillips) [inline server/authentication/authInit.js]: confused as to why you are changing this function.  Shouldn't it just be deleted as all logins should go through auth?

File: server/authentication/auth.js [hunk 2]
```diff
@@ -15,9 +16,14 @@ function authenticationMiddleware(signInService) {
...
-            const systemToken = await signInService.getClientCredentialsTokens(username);
-            res.locals.token = systemToken.token;
-            return next();
+            try {
+                const systemToken = await signInService.getClientCredentialsTokens(username);
+                res.locals.token = systemToken.token;
+                return next();
+            } catch (error) {
+                logger.error('Unable to get client credentials token for RO', error.stack);
+                return res.redirect('/logout');
```
Comment (petergphillips) [inline server/authentication/auth.js]: is the right place to redirect them?  Won't they logout, login again and then be logged out automatically?

File: server/authentication/authInit.js (hunk 1 above)
Comment (whitfield-mj) [reply server/authentication/authInit.js]: Yes everything that a user sees. It was just for instances when running locally with local signin form

File: server/authentication/auth.js (hunk 2 above)
Comment (whitfield-mj) [reply server/authentication/auth.js]: Yes. Not an ideal solution. Shouldn't really occur in the wild. Maybe this full PR is unnecessary, will close

**Pull request closed without merging**